src/main/java/io/github/moulberry/notenoughupdates/NEUOverlay.java

```diff
@@ -1143,12 +1144,14 @@ public void consume(int x, int y, int id) {
 					JsonObject item = manager.getItemInformation().get(internalname.get());
 					if (item != null) {
 						if (keyPressed == manager.keybindViewUsages.getKeyCode()) {
+							GuiItemRecipe.lastKeybindMs = System.currentTimeMillis();
```

Comment:
Couldn't we check for `Keyboard.getEventState()` instead?

Reply:
Always returns false

Reply:
yes, because this is an entire block of `if (!Keyboard.getEventState())` (which does everything on keydown (instead of keyup, like the recipe ui does). i dont really like just using a cooldown on keypresses. either properly fix the keydown event here, or just make guiitemrecipe handle keydown events as well (although i personally would not be a fan of that, that option is easier)